src/styles.scss

```diff
@@ -32,10 +62,29 @@ $abcall-theme: mat.define-theme((
 // For more information: https://material.angular.io/guide/typography#using-typography-styles-in-your-application.
 // @include mat.typography-hierarchy($abcall-theme);
 
-// Comment out the line below if you want to use the deprecated `color` inputs.
+// Comment out the line below if you want to use the deprecated color inputs.
 // @include mat.color-variants-backwards-compatibility($abcall-theme);
 
 /* You can add global styles to this file, and also import other style files */
 
-html, body { height: 100%; }
-body { margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; }
+html,
+body {
+  height: 100%;
+}
+
+h2 {
+  font-size: 20px;
+}
+a {
+  font-size: 16px;
+  text-decoration: none;
+}
+.button {
+  background-color: $blue-500;
```

Comment:
Maybe these are not necessary, if material color scheme is configured correctly.